doc/sphinx/Pacemaker_Explained/collective.rst

```diff
@@ -274,7 +275,7 @@ _____________
    |                   |                 |    single: option; clone-node-max (clone)             |
    |                   |                 |    single: clone-node-max; clone option               |
    |                   |                 |                                                       |
-   |                   |                 | If ``globally-unique`` is **true**, the maximum       |
+   |                   |                 | If ``globally-unique`` is not **false**, the maximum  |
```

Comment:
Sort of circular in the default case (`clone-node-max = 1` causes `globally-unique` to default to `false`, but the description of `clone-node-max` is still accurate). I feel like attempting to be more explicit would end up being less clear though.

Reply:
maybe some room for improvement